examples/python/2_large_model/bench_load_sglang.py

```
@@ -0,0 +1,120 @@
+        --model-path /path/to/Qwen3-30B-A3B \
+        --tp 1
+
+    # DeepSeek-V3 671B, 8 GPUs:
+    python bench_load_sglang.py \
+        --model-path /path/to/DeepSeek-V3 \
+        --tp 8
+
+Output: JSON with per-phase timing (engine_init, total).
+"""
+import argparse
+import gc
```

Flake8 (7.3.0)

[error] 'gc' imported but unused

(F401)